src/test/regress/sql/gp_toolkit.sql

```diff
@@ -66,6 +66,12 @@ select logseverity from gp_toolkit.gp_log_system where logseverity='LOG' limit 5
 select logseverity from gp_toolkit.gp_log_database where logseverity='LOG' limit 3;
 select logseverity from gp_toolkit.gp_log_master_concise where logseverity='LOG' limit 5; -- superuser only
 
+begin;
+DECLARE _psql_cursor NO SCROLL CURSOR FOR select logseverity from gp_toolkit.__gp_log_master_ext where logseverity='LOG';
```

Comment:
1. I think, that this test should be more common, because you've solved the problem for any external web table that is executed on coordinator. It would be nice to change test location too. Take a look on external_table.source. It seems, that there are appropriate for your test external tables.
2. Also, provide test description

Reply:
done

Reply:
done